Descriptive analysis report

elisabascunan opened the pull request:
This pull request will **add to main the files associated with the descriptive analysis report**:

- descriptive-analysis.qmd
- descriptive-analysis.html. 
- new folder descriptive-analysis_files, with figure_html and libs
- new folder images, with all the images from the report. 

Finally, it also **adds a new dataset** to the folder testing (regset.csv). 
This file contains the data to run the file descriptive-analysis.qmd

elisabascunan committed:
Add files via upload

elisabascunan committed:
Delete regset.csv

re-adding it to testing folder

elisabascunan committed:
Add files via upload

Commit:
First part of descriptive analysis

Commit:
edit

Commit:
part 2 of analyses

Commit:
edit to introduction

Commit:
change in outline

Commit:
final part of initial descriptive analysis

Commit:
edit

Commit:
pdf

Commit:
html

Commit:
First visualization analysis (wordclouds)

Commit:
update

elisabascunan committed:
Delete descriptive-analysis.pdf

Commit:
update

Commit:
update in text

Commit:
added analysis: location by tool and negative controls observations

Commit:
change tables for barplots

Commit:
add title

elisabascunan committed:
create new folder

elisabascunan committed:
Delete descriptive analysis/test

elisabascunan committed:
Delete Visualization_test.R

elisabascunan committed:
Add files via upload

elisabascunan committed:
Delete descriptive-analysis.qmd

elisabascunan committed:
Delete descriptive-analysis.html

elisabascunan committed:
Delete Visualization_test.R

elisabascunan committed:
Add files via upload

Commit:
visualizations

Commit:
figure

Commit:
commit

Commit:
modifications

Commit:
commit

elisabascunan committed:
Add files via upload

Commit:
Changes based on feedback from dec 1st

Commit:
Small modification from text.

elisabascunan committed:
Delete Visualizations screenIT.qmd

elisabascunan committed:
Delete Visualizations screenIT.html

elisabascunan committed:
Delete Visualizations-screenIT_files directory

elisabascunan committed:
Delete Visualizations screenIT_files directory

delwen commented:
Thanks @elisabascunan! Looks good. Regarding where to store data: I recommend creating a `data` folder to store both completed extraction sheets (both the `nc_man_check.csv` and `reg_set_man_check.csv` files). However, I also suggest to then ignore this folder in git. This means that this data folder exists locally, but doesn't get pushed to remote. It should just be a matter of creating the new folder, moving the files there, and then adding the folder `data/` in your .gitignore file (and then commit these changes). If you do this, you'll also have to change the path in your qmd where you read in the reg_set_man_check.

You can then share the two extraction sheets as CSV to others through other means so that they have it locally and be able to reproduce the analysis. We may revert on this decision and push everything to the remote, but TBD.

Thanks, great job!

elisabascunan committed:
Create new folder

New folder data for storing datasets of this project

elisabascunan committed:
Add files via upload

Uploaded negative control check extraction sheet to data folder

elisabascunan committed:
Delete data/new folder

Deleted empty file on data folder

elisabascunan committed:
Update and rename regset.csv to regset.csv

Moved extraction sheet regset.csv to new folder data/

elisabascunan committed:
Update and rename nc_man_check - nc_man_check.csv to nc_man_check.csv

Renamed file

elisabascunan committed:
Update descriptive-analysis.qmd

Modified the path in qmd to read reg_set.csv from the new folder data

elisabascunan commented:
Hi Delwen, I created the new folder and added/moved the extraction sheets there. I also modified the code on the qmd with the new path to this folder. 

With regards to your last point (adding the folder `data/` in your .gitignore file), I see that on line 6 "data/" is already on the .gitignore file: 

![Screenshot 2023-12-11 at 11 50 37](https://github.com/delwen/screenit-tool-comparison/assets/144803094/31770182-58e7-450a-8a8a-e758e45d7be6)

**I am not sure if that it is all that is needed to effectively add the folder on the .gitignore file**. If that is sufficient, then the changes are completed for the pull request.

Commit:
Deleted folder data with extraction sheet from repository.

elisabascunan commented:
The data folder with the extraction sheets was eliminated from the repository. The test_branch should be ready to merge.
